api/client/service.proto

```diff
@@ -34,10 +36,59 @@ message ExecutionResponse {
   google.rpc.Status error_detail = 7;
 }
 
+// Used to request a RequestStreamResponse.
+message RequestStreamRequest {
+  // Amount of RequestStreamResponses that should be yielded.
+  // Used to implement basic flow control by the client. Once AsyncClientImpl gets
+  // a way to apply backpressure this could probably be dropped.
+  uint64 amount = 1;
```

Comment:
can you clarify in this comment what this is the amount of?

My assumption is number of requests to specify.  Is the contract on the server side supposed to be that that's how many responses they'll stream back as output for that one request?